src/platform/forms/tests/components/FormFooter.unit.spec.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
```

Comment:
We've been trying to use Enzyme for new tests. It's not too different from skin deep: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/post-911-gib-status/tests/containers/IntroPage.unit.spec.jsx

Reply:
I will convert these tests to Enzyme, thanks for catching that!